charts/nucliadb_node/templates/node.reader.deploy.yaml

```diff
@@ -0,0 +1,155 @@
+        - effect: NoSchedule
+          key: nucliadb/node
+          operator: Equal
+          value: present
+      containers:
+        - command:
+            - node_reader
+          env:
+            - name: METRICS_HTTP_PORT
+              value: "3031"
+            - name: EXAMPLE
+              value: VALUE
+          envFrom:
+            - configMapRef:
+                name: node-config
+          image: eu.gcr.io/stashify-218417/node:b7e3f15
+          imagePullPolicy: IfNotPresent
+          livenessProbe:
+            exec:
+              command:
+                - /bin/grpc_health_probe
+                - -addr=:10001
+                - -service=nodereader.NodeReader
+            failureThreshold: 3
+            initialDelaySeconds: 2
+            periodSeconds: 10
+            successThreshold: 1
+            timeoutSeconds: 1
+          name: node-reader
+          ports:
+            - containerPort: 10001
+              name: grpc-reader
+              protocol: TCP
+            - containerPort: 3031
+              name: metrics
+              protocol: TCP
+          readinessProbe:
+            exec:
+              command:
+                - /bin/grpc_health_probe
+                - -addr=:10001
+                - -service=nodereader.NodeReader
+            failureThreshold: 10
+            initialDelaySeconds: 10
+            periodSeconds: 5
+            successThreshold: 1
+            timeoutSeconds: 1
+          securityContext:
+            privileged: true
+          terminationMessagePath: /dev/termination-log
+          terminationMessagePolicy: File
+          volumeMounts:
+            - mountPath: /data
+              name: node-pv
+{{- if $values.tracing.enabled }}
+        - name: jaeger-agent
+          image: "jaegertracing/jaeger-agent:{{ $values.tracing.jaegerAgentTag }}"
+          imagePullPolicy: IfNotPresent
+          ports:
+            - containerPort: 5775
+              name: zk-compact-trft
+              protocol: UDP
+            - containerPort: 5778
+              name: config-rest
+              protocol: TCP
+            - containerPort: 6831
+              name: jg-compact-trft
+              protocol: UDP
+            - containerPort: 6832
+              name: jg-binary-trft
+              protocol: UDP
+            - containerPort: 14271
+              name: admin-http
+              protocol: TCP
+          args:
+            - --reporter.grpc.host-port=dns:///{{ $values.tracing.jaegerCollectorHost }}:{{ $values.tracing.jaegerCollectorGrpcPort }}
+            - --reporter.type=grpc
+{{- end }}
+      dnsPolicy: ClusterFirst
+      restartPolicy: Always
+      schedulerName: default-scheduler
+      securityContext: {}
+      volumes:
+        - name: nats-creds
+          secret:
+            defaultMode: 420
+            secretName: nats-nucliadb-creds
+        - name: regional-nats-creds
+          secret:
+            defaultMode: 420
+            secretName: nats-regional-creds
```

Comment:
why are these needed?

Reply:
oh, it's just copy paste from existing deploy. I can remove